go.mod

```diff
@@ -1,16 +1,14 @@
-module github.com/chaos-mesh/k8s_dns_chaos
-
```

Comment:
Hi @elhayef , thanks for your contribution.

But this PR removes some instructions in go.mod and the file go.sum, which would break the project structure. 

please fix it